ee/clickhouse/queries/funnels/funnel_strict.py

```diff
@@ -16,7 +16,18 @@ def get_step_counts_query(self):
 
         max_steps = len(self._filter.entities)
 
-        partition_select = self.get_partition_cols(1, max_steps)
+        formatted_query = self.get_step_counts_without_aggregation_query()
```

Comment:
this pattern extends well to all types of orderings: First we get the base un-aggregated query, then perform some aggregation on it.

Split it into these two components, because some visualisations need the un-aggregated version (trends), while others need the aggregated version (time to convert)